session/session.go

```diff
@@ -1271,6 +1336,11 @@ func CreateSession(store kv.Storage) (Session, error) {
 	}
 	privilege.BindPrivilegeManager(s, pm)
 
+	bm := &bindinfo.BindManager{
+		GlobalAccessor: s,
```

Comment:
I don't think it is a proper way to use the current session to manage bind info. For example, if the current session is already in a transaction, then the `begin` in https://github.com/pingcap/tidb/pull/9846/files#diff-928cb614ae092ed4c6f349f4c4c991e4R882 would 
auto commit the transaction, which would lead to unexpected behavior.